core/templates/dev/head/player/ProgressDotsDirective.js

```diff
@@ -43,7 +43,6 @@ oppia.directive('progressDots', [function() {
             return;
           } else if (newValue === oldValue + 1) {
             $scope.dots.push({});
-            playerPositionService.setActiveCardIndex($scope.dots.length - 1);
```

Comment:
I deleted this, because it was triggering setActiveCardIndex twice (the second was via calback in PlayerPositionService), which is not good for performance. Was there any reason?